snow/consensus/snowball/parameters.go

```diff
@@ -103,16 +99,14 @@ func (p Parameters) Verify() error {
 		return fmt.Errorf("%w: alphaPreference = %d, alphaConfidence = %d: fails the condition that: alphaPreference <= alphaConfidence", ErrParametersInvalid, p.AlphaPreference, p.AlphaConfidence)
 	case p.K < p.AlphaConfidence:
 		return fmt.Errorf("%w: k = %d, alphaConfidence = %d: fails the condition that: alphaConfidence <= k", ErrParametersInvalid, p.K, p.AlphaConfidence)
-	case p.BetaVirtuous <= 0:
-		return fmt.Errorf("%w: betaVirtuous = %d: fails the condition that: 0 < betaVirtuous", ErrParametersInvalid, p.BetaVirtuous)
-	case p.BetaRogue == 3 && p.BetaVirtuous == 28:
-		return fmt.Errorf("%w: betaVirtuous = %d, betaRogue = %d: fails the condition that: betaVirtuous <= betaRogue\n%s", ErrParametersInvalid, p.BetaVirtuous, p.BetaRogue, errMsg)
-	case p.BetaRogue < p.BetaVirtuous:
-		return fmt.Errorf("%w: betaVirtuous = %d, betaRogue = %d: fails the condition that: betaVirtuous <= betaRogue", ErrParametersInvalid, p.BetaVirtuous, p.BetaRogue)
+	case p.Beta <= 0:
+		return fmt.Errorf("%w: beta = %d: fails the condition that: 0 < beta", ErrParametersInvalid, p.Beta)
+	case p.AlphaConfidence == 3 && p.AlphaPreference == 28:
+		return fmt.Errorf("%w: alphaConfidence = %d, alphaPreference = %d: fails the condition that: alphaPreference <= alphaConfidence\n%s", ErrParametersInvalid, p.AlphaConfidence, p.AlphaPreference, errMsg)
```

Comment:
q: we where testing `p.BetaRogue` and `p.BetaVirtuous` with. a similar condition. Why we replaced them with `p.AlphaConfidence` instead of dropping this?

Reply:
I wanted to keep `errMsg` 😉